static/sass/style.scss

```diff
@@ -2055,12 +2055,45 @@ $colors: $blue, $psf, $yellow, $green, $purple, $red;
 
 /* ! ===== User profile pages and sign up forms ===== */
 .user-profile-controls {
-    border-bottom: 1px solid $default-border-color;
     margin-bottom: 2em;
 
-    li {
-        @include pe-inline-block();
-        margin-right: .75em;
+    .menu {
+      display: flex;
+      width: 100%;
+    }
+
+    div.section {
+      padding: 0.25em 1em;
+
+      &:hover {
+          .section-links {
+              display: inline;
+          }
+      }
+
+      span {
+        @extend %button-style-yellow;
```

Comment:
<img width="586" alt="Screen Shot 2021-04-13 at 3 44 23 PM" src="https://user-images.githubusercontent.com/1200832/114611569-2f12db80-9c6f-11eb-9246-d7e02301a5ea.png">
This button styling doesn't seem to be working for me locally.